templates/frontend/objects/issue_toc.tpl

```diff
@@ -17,109 +17,92 @@
+{if !$issue->getPublished()}
+	{include file="frontend/components/notification.tpl" type="warning" messageKey="editor.issues.preview"}
+{/if}
 
-	{* Published date *}
-	{if $issue->getDatePublished()}
-		<div class="published">
-			<span class="date_label">
-				{translate key="submissions.published"}
-			</span>
-			<span class="date_format">
-					{$issue->getDatePublished()|date_format:$dateFormatLong}
-			</span>
-		</div>
-	{/if}
-
-	{* Indicate if this is only a preview *}
-	{if !$issue->getPublished()}
-		{include file="frontend/components/notification.tpl" type="warning" messageKey="editor.issues.preview"}
-	{/if}
-
-	{* Issue introduction area above articles *}
-	{if $issue->hasDescription() || $issue->getLocalizedCoverImageUrl()}
-		<div class="issue_heading">
-			{* Description *}
-			<div class="flex_container description_cover">
-				{if $issue->hasDescription()}
-					<div class="description">
-						<h2 class="description_label">{translate key="plugins.themes.classic.issueDescription"}</h2>
-						{assign var=issueDescription value=$issue->getLocalizedDescription()|strip_unsafe_html}
-						{if $issueDescription|strlen < 800}
-							<div class="description_text">
-								{$issueDescription}
-							</div>
-						{elseif $requestedPage|escape !== "issue"}
-							<div class="description_text">
-								{$issueDescription|substr:0:800|mb_convert_encoding:'UTF-8'|replace:'?':''}<span
-										class="three_dots">...</span>
-								<a class="more_button"
-								   href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
-									{translate key="plugins.themes.classic.more"}
-								</a>
-							</div>
-						{else}
-							<div class="description_text">
-								{$issueDescription}
-							</div>
-						{/if}
+{* Issue introduction area above articles *}
+{if $issue->hasDescription() || $issue->getLocalizedCoverImageUrl()}
+	{* Description *}
+	<section class="flex_container description_cover">
+		{if $issue->hasDescription()}
+			<div class="description">
+				<h3 class="description_label">{translate key="plugins.themes.classic.issueDescription"}</h3>
+				{assign var=issueDescription value=$issue->getLocalizedDescription()|strip_unsafe_html}
+				{if $issueDescription|strlen < 800}
+					<div class="description_text">
+						{$issueDescription}
 					</div>
-				{/if}
-
-				{* Issue cover image *}
-				{assign var=issueCover value=$issue->getLocalizedCoverImageUrl()}
-				{if $issueCover}
-					<div class="issue_cover_block{if !$issue->hasDescription()} align-left{/if}">
-						<a href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
-							<img class="cover_image"
-							     src="{$issueCover|escape}"{if $issue->getLocalizedCoverImageAltText() != ''} alt="{$issue->getLocalizedCoverImageAltText()|escape}"{/if}>
+				{elseif $requestedPage|escape !== "issue"}
+					<div class="description_text">
+						{$issueDescription|substr:0:800|mb_convert_encoding:'UTF-8'|replace:'?':''}<span
```

Comment:
Perhaps a note for @Vitaliy-1: Smarty includes a `truncate` method which you might be able to use here. 

https://www.smarty.net/docs/en/language.modifier.truncate.tpl

The benefit is that it tries to cut off at a word, rather than the exact character limit. But I don't know if it accounts for the character handling you're doing here.

Reply:
I've used the `truncate` method in one of the new themes. It has several benefits but still has some problems regarding non-Latin symbols.

Reply:
From https://www.smarty.net/docs/en/charset.tpl:
```
Smarty's internals and core plugins are truly UTF-8 compatible since Smarty 3.1. To achieve unicode compatibility, the Multibyte String PECL is required. Unless your PHP environment offers this package, Smarty will not be able to offer full-scale UTF-8 compatibility.
```
If you're not seeing mbstring-safe behavior, check to make sure you've got mbstring support in your PHP?

Reply:
:+1:

There are a couple of places that we use `|truncate` in core. Could you document the problems in an issue and describe your solution? Maybe we can come up with a `|symbol_safe_truncate` function to use.

Reply:
Thanks, will recheck this.

Reply:
Tested with Cyrillic (Ukrainian), works fine!